pypesto/optimize/optimizer.py

```diff
@@ -555,3 +560,49 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class CmaesOptimizer(Optimizer):
+    """
+    Global optimization using cma-es.
+    """
+
+    def __init__(self, options: Dict = None):
+        super().__init__()
+
+        if options is None:
+            options = {'maxiter': 10000}
+        self.options = options
+
+    @fix_decorator
+    @time_decorator
+    @history_decorator
+    def minimize(
+            self,
+            problem: Problem,
+            x0: np.ndarray,
+            id: str,
+            history_options: HistoryOptions = None,
+    ) -> OptimizerResult:
+        lb = problem.lb
+        ub = problem.ub
+        sigma0 = 0.25 * np.median(ub)
```

Comment:
This hyper parameter will probably have a Hughe influence on the results and is hidden somewhere in the code and hardcoded. :/

Is there a rationale, why you chose `0.25 * np.median(ub)`?

Reply:
The documentation (http://cma.gforge.inria.fr/apidocs-pycma/cma.evolution_strategy.html#fmin) suggests sigma0 to be 0.25 of the search domain width

Reply:
Jakob means to add the 0.25 as a parameter to the class, so that you can then here have `self.THAT_PARAMETER * np.median(ub)` or sth like that.

Reply:
Done

Reply:
Done